test/indexer/data/b7fc737e-9b7b-4800-8977-fe7c94e131df.metadata.json

```diff
@@ -56,7 +56,7 @@
             }
         ],
         "organ": {
-            "text": "Brain",
+            "text": "Brain 1",
```

Comment:
Should there be an assertion matching this change? If not should the change be removed?

Reply:
Yes, this bundle combines 3 specimens into 1 cell suspension and will be good test to verify how the specimen organ values are handled when they are different. I will add the missing test.